pkg/util/jsonhelper.go

```diff
@@ -17,22 +17,24 @@ package util
 import (
 	"bytes"
 	"encoding/json"
-	"strings"
-
 	"github.com/sirupsen/logrus"
+	"strings"
 )
 
 // MinifyJSON is used to compact the json bytes (e.g. removing whitespaces and new lines character)
-func MinifyJSON(jsonBytes []byte) string {
+func MinifyJSON(jsonBytes []byte, from, details string) string {
```

Comment:
Actually, I don't like the method abstract here:
1. minifyJSON is a purely least unit, it shouldn't involve any business logic, don't add extra params like details, don't log in the method, instead throw the error directly
2. It's the consumer's responsibility to log and track the detail